YoCode/GitCheck.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+using System.IO;
+
+namespace YoCode
+{
+    public class GitCheck
+    {
+        //needs to be fixed
+        public string REPOSITORY_PATH = @"C:\Users\ukmzil\source\repos\Tests Sent by People\Real\simon-jones";
+
+        public string Output { get; set; }
+        public string LastAuthor { get; set; }     
+        public bool GitUsed { get; set; }
+
+        public List<string> hostDomains = new List<string>();
+
+        public bool ExecuteTheCheck()
```

Comment:
Might be a good idea to add constructor, as this method is always called when GitCheck object is created to populate all data fields.